generated/nidcpower/nidcpower/session.py

```diff
@@ -118,7 +118,7 @@ class _SessionBase(object):
     _is_frozen = False
 
     _active_advanced_sequence = _attributes.AttributeViString(1150074)
-    '''Type: str
+    '''Type: default_str
```

Comment:
These are all wrong

Reply:
Lost fix when moving to new PR. Fixed. Again.